create_model.py

```diff
@@ -1,23 +1,28 @@
 from text2emoji.models.grid_search_model import GridSearchModel
+from text2emoji.models.test_model import test_best_model
 
 
-def find_best_model():
+def find_best_model(model_type):
     hyperparameters = {
-        "n_dimensions": [10, 25, 50],
-        "n_layers": [1, 2],
-        "n_neurons": [20, 50],
-        "optimizer_type": ["adam", "sgd"],
-        "learning_rate": [0.01, 0.001],
-        "epochs": [5, 10, 20],
+        "dimensionality_reduction": ["none"],
```

Comment:
Great that you added the skipping some steps such as dim reduction, it's quite interesting that it reduces the performance